issuer-front-end/src/SelectNetwork.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+import React, { Dispatch, SetStateAction } from 'react';
+
+interface ConnectionProps {
+    setIsTestnet: Dispatch<SetStateAction<boolean | undefined>>;
+}
+
+export default function SelectNetwork(props: ConnectionProps) {
+    const { setIsTestnet } = props;
+
+    const changeDropDownHandler = () => {
+        const e = document.getElementById('write') as HTMLSelectElement;
+        const sel = e.selectedIndex;
+        const { value } = e.options[sel];
```

Comment:
It should be possible to get the value out in a nicer way than this. Something like:

```suggestion
    const changeDropDownHandler = (e: ChangeEvent) => {
        const value = e.target.value; // can't remember exactly where it's located
```